packages/bot-web-ui/src/components/dashboard/dashboard-component/__tests__/dashboard-component.spec.tsx

```diff
@@ -60,7 +47,15 @@ export const mocked_props = {
 
 describe('<Dashboard />', () => {
```

Comment:
why do you write here that you are testing <Dashboard, but you test here <UserGuide component?
is it expected?

Reply:
file still here